Rakefile

```diff
@@ -176,8 +176,8 @@ task :xcode => [:dependencies] do
 end
 
 desc "Run all code generation tasks"
-task :generate do 
-  ["Networking", "Yosemite", "WooCommerce"].each { |prefix| 
+task :generate do
+  ["Networking", "Yosemite", "WooCommerce"].each { |prefix|
```

Comment:
This is the same thing that happened with https://github.com/woocommerce/woocommerce-android/pull/2568, but in this instance the change was really minimal.

If in that PR we'll make a decision regarding `.editorconfig` I might open an issue suggesting to add it to this repo, too.